docs/_snippets/features/image-style-custom.js

```diff
@@ -12,16 +12,16 @@ ClassicEditor
 		image: {
 			styles: [
 				// This option is equal to a situation where no style is applied.
-				'imageStyleFull',
+				'imageStyle:full',
```

Comment:
It doesn't make sense to repeat the `imageStyle:` prefix in `image.styles` config. I'll remove it.